.gitignore

```diff
@@ -1,8 +1,245 @@
-*.nupkg
-*.snupkg
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Keep `.gitignore` simple. Revert all changes here and ignore only what necessary.